kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/DeltaLogFile.java

```diff
@@ -40,6 +41,10 @@ public static DeltaLogFile forCommitOrCheckpoint(FileStatus file) {
     if (FileNames.isCommitFile(fileName)) {
       logType = LogType.COMMIT;
       version = FileNames.deltaVersion(fileName);
+    } else if (FileNames.isLogCompactionFile(fileName)) {
+      logType = LogType.LOG_COMPACTION;
+      // use end version, similar to a checkpoint
```

Comment:
nit - maybe document this either as part of `getVersion` or `LOG_COMPACTION` enum? Just to make it as obvious as possible

Reply:
Also should we rename this function name? `forCommitOrCheckpoint`?